src/Dev/TestKernel.php

```diff
@@ -59,6 +59,9 @@ public function setIgnoredCIConfigs(array $ciConfigs): self
         return $this;
     }
 
+    /**
+     * @deprecated This method will be removed in CMS 5
```

Comment:
Add version number, which should not be 5.0.0 (the notice below will need its version updated).
Also provide a more standardised message here and in the notice.

Reply:
Updated